fixtures/invalid_a_field.igc

```diff
@@ -0,0 +1,10488 @@
+AXGD Flymaster F1, V1.21, S/N 3556
```

Comment:
I'm not quite sure why this line is considered invalid. The `A` record regular expression should accept everything that starts with `A` and is followed by at least three characters.

Maybe the A record parser has a bug? 🤔